lib/prompt.js

```diff
@@ -5,14 +5,25 @@ const { promisify } = require('util');
 const read = promisify(require('read'));
 
 async function prompt(message, options) {
+    let value;
+
     // Read input
-    let value = await read({
-        prompt: message,
-        silent: options.silent,
-        replace: options.replace,
-        input: options.input,
-        output: options.output,
-    });
+    // Manage timeout
+    try {
+        value = await read({
+            prompt: message,
+            silent: options.silent,
+            replace: options.replace,
+            input: options.input,
+            output: options.output,
+            timeout: options.timeout,
+        });
+    } catch (err) {
+        if (err.message !== 'timed out' || !options.default || !options.useDefaultOnTimeout) {
```

Comment:
I think this condition won’t work if default is an empty string, since you are using !options.default. Maybe you should compare against undefined just like we are doing in line 35?

Reply:
You are right, I fixed it and added the unit test, nice catch!